pwiz_tools/Skyline/TestPerf/SmallMolLibrariesTutorialTest.cs

```diff
@@ -0,0 +1,431 @@
+            //   �	Click No on the form asking if you want to save the current settings.
+            RunUI(() => SkylineWindow.ResetDefaultSettings());
+            MaximizeSkylineWindow(); // Set the Skyline window large enough to accomodate all windows for screenshots
+
+            var doc = SkylineWindow.Document;
+
+            //   �	On the Settings menu, click Transition Settings.
+            //   �	Click the Filter tab.
+            var transitionSettingsUI = ShowDialog<TransitionSettingsUI>(() =>
+                SkylineWindow.ShowTransitionSettingsUI(TransitionSettingsUI.TABS.Filter));
+
+            //   �	This data was collected in negative ionization mode so [M+H] and[M +] can be removed from the Precursor Adducts and Fragment Adducts fields. However, they are harmless if left as is since the library we will use has only negative ion mode entries.
+            //   �	In the Precursor Adducts field, enter �[M-H], [M+HCOO], [M+CH3COO]�. 
+            //   �	In the Fragment Adducts field, enter �[M-]�.
+            RunUI(() =>
+            {
+                transitionSettingsUI.SmallMoleculePrecursorAdducts = "[M-H], [M+HCOO], [M+CH3COO]";
+                transitionSettingsUI.SmallMoleculeFragmentAdducts = "[M-]";
+                transitionSettingsUI.SmallMoleculeFragmentTypes = "f, p";
+            });
+            //   �	The Transition Settings form should now look like this:
+            PauseForScreenShot("Transition Settings: Filter", 4);
+
+            RunUI(() =>
+            {
+                //   �	Click the Full-Scan tab in the Transition Settings form.
+                transitionSettingsUI.SelectedTab = TransitionSettingsUI.TABS.FullScan;
+                //   �	 In the MS1 filtering section, set the Isotope peaks included field to Count.
+                transitionSettingsUI.PrecursorIsotopesCurrent = FullScanPrecursorIsotopes.Count;
+                //   �	Set the Precursor mass analyzer field to TOF.
+                transitionSettingsUI.PrecursorMassAnalyzer = FullScanMassAnalyzerType.tof;
+                //   �	Enter �20,000� in the Resolving power field.
+                transitionSettingsUI.PrecursorRes = 20000;
+                //   �	In the MS/MS filtering section, set the Acquisition method field to DIA.
+                transitionSettingsUI.AcquisitionMethod = FullScanAcquisitionMethod.DIA;
+                //   �	Set the Isolation scheme field to All Ions.
+                transitionSettingsUI.IsolationSchemeName = IsolationScheme.SpecialHandlingType.ALL_IONS;
+                //   �	Enter �20,000� in the Resolving power field.
+                transitionSettingsUI.ProductRes = 20000;
+                //   �	Check Use high-selectivity extraction.
+                transitionSettingsUI.UseSelectiveExtraction = true;
+            });
+
+            //   The Transition Settings form should look like this:
+            PauseForScreenShot("Transition Settings - Full-Scan", 5);
```

Comment:
Use the IForView type in this pause for the Transition Settings - Full-Scan tab

Reply:
right, thanks.